docs/products/opensearch/howto/opensearch_alerting.rst

```diff
@@ -0,0 +1,142 @@
+
+
+Create a destination
+********************
+
+1. Click the **Destination** tab -> **Add destination**
+   
+2. **Name** -> ``slack-test``
+
+   **Type** -> ``Slack``
+
+   **Settings** **Webhook URL:** -> ``https://your_slack_webhook_URL``
+
+.. note::
+   Destination Type can be: ``Amazon Chime``, ``Slack``, ``Custom webhook`` or ``Email``
+
+.. important::
+   When using email you need to have a SMTP server configured for a valid domain to deliver email notifications
+
+Create a monitor
+****************
+
+1. Click the **Monitors** tab -> **Create monitor**
+
+2. **Monitor details**
+   
+   **Monitor name** -> ``High CPU Monitor``
+
+   **Monitor name** -> ``Per query monitor``
+   
+   **Monitor defining method** -> ``Visual editor`` 
+
+   **Schedule** **Frequency** -> ``By interval``
+
+   **Run every** -> ``1`` ``Minutes``
+
+.. note::
+   Schedule Frequency can be `By internal`, `Daily`, `Weekly`, `Monthly`, `Custom CRON expression`
+
+3. **Data source** 
+ 
+   **index** -> ``sample-host-health``
+
+   **Time field** -> ``timestamp``
+
+4. **Query**
+
+   **Metrics** -> **Add metric** 
+
+   **Aggregation** ``avrerage()`` **Field** ``cpu_usage_percentage``
+
+   **Time range for the last** ``3`` ``minutes``
+
+Create a trigger
+****************
+
+1. **Add trigger**
+
+   **Trigger name** -> ``high_cpu``
+
+   **Severity level** -> ``1 (Highest)``
+
+   **Trigger condition** ``IS ABOVE`` ``75``
+
+.. note::
+   You can see a visual graph below trigger with the index data and the trigger condition you have defined as a red line
+
+2. **Actions**
+     
+   **Action name** -> ``slack``
+
+   **Destination** -> ``slack-test``
+
+   **Message subject** -> ``High CPU Test Alert``
+
+.. note::
+   Multiple Actions can be defined, in this example we will define one action to send notification to destination we have defined in step 4
+
+Alert Message
+*************
+
+**Message** can be adjusted as needed, check **Message Preview** to see the sample and use **Send test message** to validate notification delivery
+
+Click on **Create** and your monitor is ready!
+
+* For further details on `alerting monitors configuration <https://opensearch.org/docs/latest/monitoring-plugins/alerting/monitors/>`_
+
+.. _Code:
+
+Create programmatically
+***********************
+
+Monitors can be created by ``POST`` a JSON to OpenSearch API 
+
+``https://username:password@os-name-myproject.aivencloud.com:24947/_plugins/_alerting/monitors``
+
+The required JSON request format can be found in `OpenSearch Alerting API documentation <https://opensearch.org/docs/latest/monitoring-plugins/alerting/api/#create-query-level-monitor>`_
+
+
+The following example code is for creating the same CPU alert monitor above programmatically.
+
+Save the JSON below into ``cpu_alert.json`` 
+
+.. literalinclude:: /code/products/opensearch/cpu_alert.json
+  :language: JSON
+
+Use ``curl`` to create the alert
```

Comment:
Where does the URL come from? This looks like magic, can we add a few more words?